docs/content_management/create_edit_content_items.md

```diff
@@ -197,23 +197,26 @@ Contact your website administrator about the details of using them.
 
 ### AI Assistant
 
-If the [AI Actions](ai_actions.md) LTS update has been installed and configured in your application, when writing content, the **Improve text** button is visible in the toolbox.
+If the [AI Actions](ai_actions.md) LTS update has been installed and configured in your application, when writing content, the **Improve text** button is visible in online editor's toolbox.
```

Comment:
```suggestion
If the [AI Actions](ai_actions.md) LTS update has been installed and configured in your application, when writing content, the **Improve text** button is visible in the online editor's toolbox.
```

Reply:
Added in https://github.com/ibexa/documentation-user/pull/333/commits/7b8459c419ed63a5a9b587fd0711b7419100f344 , thanks!